packages/data-store/src/utils/digitalCredential/MappingUtils.ts

```diff
@@ -0,0 +1,112 @@
+import { AddDigitalCredentialArgs } from '../../types/digitalCredential/IAbstractDigitalCredentialStore'
+import { DigitalCredentialEntity } from '../../entities/digitalCredential/DigitalCredentialEntity'
+import {
+  CredentialMapper,
+  decodeSdJwtVc,
+  DocumentFormat,
+  IVerifiableCredential,
+  IVerifiablePresentation,
+  SdJwtDecodedVerifiableCredentialPayload,
+} from '@sphereon/ssi-types'
+import {
+  CredentialDocumentFormat,
+  CredentialType,
+  DigitalCredential,
+  NonPersistedDigitalCredential,
+} from '../../types/digitalCredential/digitalCredential'
+import { computeEntryHash } from '@veramo/utils'
+import { createHash } from 'crypto'
+
+function determineCredentialType(raw: string): CredentialType {
+  if (CredentialMapper.hasProof(raw)) {
+    if (CredentialMapper.isCredential(raw)) {
+      return CredentialType.VC
+    } else if (CredentialMapper.isPresentation(raw)) {
+      return CredentialType.VP
+    }
+  } else {
+    if (CredentialMapper.isCredential(raw)) {
+      return CredentialType.C
+    } else if (CredentialMapper.isPresentation(raw)) {
+      return CredentialType.P
+    }
+  }
+  throw new Error(`Couldn't determine the type of the credential: ${raw}`)
+}
+
+function determineCredentialDocumentFormat(documentFormat: DocumentFormat): CredentialDocumentFormat {
+  switch (documentFormat) {
+    case DocumentFormat.JSONLD:
+      return CredentialDocumentFormat.JSON_LD
+    case DocumentFormat.JWT:
+      return CredentialDocumentFormat.JWT
+    case DocumentFormat.SD_JWT_VC:
+      return CredentialDocumentFormat.SD_JWT
+    default:
+      throw new Error(`Not supported document format: ${documentFormat}`)
+  }
+}
+
+function getExpiresAt(uniformDocument: IVerifiableCredential | IVerifiablePresentation | SdJwtDecodedVerifiableCredentialPayload): Date | undefined {
```

Comment:
See remark elsewhere about new names in VCDM 2; suggest to start using these

Reply:
changed this to validFrom